lib/error-reporter.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  reportError: (message, context) => {
+    if (!window || typeof CustomEvent === 'undefined') return
```

Comment:
This is only needed because `CustomEvent` is not available in NodeJS, so the tests fail. Ideas are welcome :)